docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
 | Upsample | No |
 | Where | No |
 | Xor | No |
+
+
+## PaddlePaddle\* Supported Operators
+
+Standard PaddlePaddle(paddlepaddle>=2.1)\* Operators:
+
+| Operator Name in PaddlePaddle\*| Limitations|
+| :----------| :----------|
+| adpative_pool2d | Only support 'NCHW' input data_layout |
+| arg_max | No |
+| assign_value | No |
+| batch_norm | No |
+| bilinear_interp | Only supports the same data_layout |
+| bilinear_interp_v2 | Only supports the same data_layout |
+| bmm | No |
+| cast | No |
+| clip | No |
+| concat | No |
+| conv2d | Only supports 'NCHW' layout |
+| depthwise_conv2d | Only supports 'NCHW' layout |
+| deformable_conv | No |
+| elementwise_add | No |
+| elementwise_div | No |
+| elementwise_max | No |
+| elementwise_min | No |
+| elementwise_mul | No |
+| elementwise_pow | No |
+| elementwise_sub | No |
+| equal | No |
+| expand_v2 | No |
+| fill_constant_batch_size_like | Only supports 'FP32' data type |
+| fill_constant | No |
+| flatten_contiguous_range | No |
+| greater_equal | No |
+| hard_sigmoid | No |
+| hard_swish | No |
+| leaky_relu | No |
+| log | No |
+| logical_not | No |
+| matmul | No |
+| matrix_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| max_pool2d_with_index | No |
+| mul | No |
+| multiclass_nms | Only supports IE CPU plugin with 'number of selected boxes' static shape(eg: min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)) |
+| nearest_interp | Only supports the same data_layout |
+| nearest_interp_v2 | Only supports the same data_layout |
```

Comment:
Same should be NCHW layout

Reply:
done, changed  the interp ops' limitation description.